highlighting.go

```diff
@@ -112,6 +112,12 @@ type Config struct {
 	// Supported styles are defined under https://github.com/alecthomas/chroma/tree/master/formatters.
 	Style string
 
+	// If set, will try to guess syntax if no language is provided.
+	// If the guessing fails, we will fall back to a text lexer.
+	// Note that while Chroma's API supports syntax guessing, the implementation
+	// is not there yet, so you will currently always get the basic text lexer.
+	GuessSyntax bool
```

Comment:
`GuessLanguage` is more preferable because CommonMark defines first space separated term of the info string in fenced codes is a "language" .